src/sage/combinat/k_regular_sequence.py

```diff
@@ -1,3 +1,4 @@
+# sage.doctest: optional - sage.combinat sage.modules
```

Comment:
why not adding `sage.symbolic` as it is necessary for many doctests ?

Reply:
I agree. Although some of the doctests can be tested without it, `sage.symbolic` is needed whenever recurrence equations are used as input, and that seems to be a main functionality of this module.
Moreover, nothing depends on the pair of modules `sage.combinat.{k_regular_sequence,recognizable_series}`, so there is probably little value in testing a subset of this functionality separately.